File: DigitalisationSolde/src/main/java/com/digitalisationSolde/model/Document3001.java
package com.digitalisationSolde.model;

import jakarta.persistence.Entity;
import lombok.Data;

import java.util.*;
@Entity
@Data
public class Document3001 extends Document {
   private int tauxRappel;
   private int pointsRappel;
   private Date dateDebutRappel;
   private Date dateFinRappel;
   private int nombreJoursRappel;
   private int codePoste;
   private int montant;

}

File: DigitalisationSoldeFrontend/src/app/_interfaces/document3001.ts
import { IDocument } from "./document";

export interface IDocument3001 extends IDocument{
    tauxRappel: number;
    pointsRappel: number;
    dateDebutRappel: Date;
    dateFinRappel: Date;
    nombreJoursRappel: number;
    codePoste: number;
    montant: number;
}
